### PR TITLE
[add] Basic Architecture Information added

### PR DESCRIPTION
**Pull request by atb00ker**
Added a short explainer for the architecture.

Added the following page:

architecture

### CODE REVIEW

File: user/architecture.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+A `larger version of the architecture image <https://miro.com/app/board/o9J_kvt3edQ=/?moveToWidget=3074457361507412175&cot=14>`_ and more detailed architectures of the modules can be found on Miro.
+
+- Server Side:
+   - OpenWISP (You have 2 main options):
+       - Ansible-OpenWISP2 (Recommended): Primary method to install
+         OpenWISP. Find `ansible-openwisp2 architecture details on GitHub <https://github.com/openwisp/ansible-openwisp2#architecture>`_.
+       - Docker-OpenWISP (In-Alpha): The future for installation of
+         OpenWISP. Find `docker-openwisp architecture details on GitHub <https://github.com/openwisp/docker-openwisp#architecture>`_.
+   - WiFi Login Pages (Optional): Configurable captive page for
+     public/private WiFi services providing login, sign up etc.
```
Comment (nemesisdesign) [inline user/architecture.rst:19]: would be good to link https://github.com/openwisp/ansible-openwisp-wifi-login-pages

File: user/architecture.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- Network Device Side:
+   - OpenWISP Config: Package that helps OpenWRT devices to connect
```
Comment (nemesisdesign) [inline user/architecture.rst:21]: would be good to link openwisp-config

**Review by nemesisdesign — COMMENTED** [review]
In place of Coova-chilli I'd put Captive portal (Coova-chilli, pfsense, opnsense, etc.).

Let's not forget that RADIUS and captive portal make sense only for WiFi / network services which require authentication.

Some companies use OpenWISP only for configurations because they have the need to provision a large number of devices.

File: user/architecture.rst (hunk 1 above)
Comment (atb00ker) [reply user/architecture.rst:19]: Done.

File: user/architecture.rst (hunk 2 above)
Comment (atb00ker) [reply user/architecture.rst:23]: Done.

**Comment by nemesisdesign** [comment]
Thanks Ajay, I used your work as a base, rebased on master and improved it in https://github.com/openwisp/openwisp2-docs/pull/157.

**Pull request closed without merging**